src/main/java/com/example/codebase/domain/agora/entity/Agora.java

```diff
@@ -0,0 +1,191 @@
+
+    private LocalDateTime updatedTime;
+
+    @ManyToOne
+    @JoinColumn(name = "author_id")
+    private Member author;
+
+    @Getter
+    @Builder.Default
+    @OneToMany(mappedBy = "agora", cascade = CascadeType.ALL)
+    private List<AgoraMedia> medias = new ArrayList<>();
+
+    @Builder.Default
+    @OneToMany(mappedBy = "agora")
+    private List<AgoraParticipant> participants = new ArrayList<>();
+
+    @Builder.Default
+    @OneToMany(mappedBy = "agora")
+    private List<AgoraOpinion> opinions = new ArrayList<>();
+
+    public static Agora from(AgoraCreateDTO dto) {
+        return Agora.builder()
+                .title(dto.getTitle())
+                .content(dto.getContent())
+                .agreeText(dto.getAgreeText())
+                .naturalText(dto.getNaturalText())
+                .disagreeText(dto.getDisagreeText())
+                .agreeCount(0)
+                .naturalCount(0)
+                .disagreeCount(0)
+                .participantCount(0)
+                .isAnonymous(dto.getIsAnonymous())
+                .createdTime(LocalDateTime.now())
+                .build();
+    }
+
+    public static Agora of(AgoraCreateDTO dto, Member member) {
+        Agora agora = from(dto);
+        agora.setMember(member);
+        return agora;
+    }
+
+    private void setMember(Member member) {
+        this.author = member;
+        member.addAgora(this);
+    }
+
+    public Boolean isAuthorUsername(String username) {
+        return this.author.getUsername().equals(username);
+    }
+
+    /**
+     * 아고라 삭제 (isDeleted = true) SOFT DELETE
+     */
+    public void delete() {
+        this.isDeleted = true;
+    }
+
+    /**
+     * 아고라 삭제 여부 확인
+     * E2E 테스트 시 동일 트랜잭션 내에서 삭제된 아고라를 다시 조회하는 경우
+     * 영속성 컨텍스트에서 1차 캐싱된 엔티티를 찾기 때문에 @Where(clause = "is_deleted = 0") 조건이 적용되지 않음
+     * 따라서 이를 해결하기 위해서 삭제 여부를 확인하는 메서드를 추가함
+     */
+    public void isDeleted() {
+        if (this.isDeleted) {
+            throw new NotFoundException("존재하지 않는 아고라입니다");
+        }
+    }
+
+    public void addMedia(AgoraMedia agoraMedia) {
+        this.medias.add(agoraMedia);
+    }
+
+    public void addParticipant(AgoraParticipant agoraParticipant) {
+        this.participants.add(agoraParticipant);
+        this.participantCount = this.participants.size();
+    }
+
+    public void addOpinion(AgoraOpinion agoraOpinion) {
+        this.opinions.add(agoraOpinion);
+    }
+
+    public void update(AgoraUpdateDTO dto) {
+        this.title = dto.getTitle();
+        this.content = dto.getContent();
+        this.agreeText = dto.getAgreeText();
+        this.naturalText = dto.getNaturalText();
+        this.disagreeText = dto.getDisagreeText();
+        this.isAnonymous = dto.getIsAnonymous();
+        this.updatedTime = LocalDateTime.now();
+    }
+
+    public boolean isCorrectVoteText(String vote) {
+        return this.agreeText.equals(vote) || this.disagreeText.equals(vote) || this.naturalText.equals(vote);
```

Comment:
중복되는 부분은 앞에 두고 비교하고자 하는 부분을 인자로 넣으면 가독성이 좋아질 것 같습니다.
```suggestion
        return vote.equals(this.agreeText) || vote.equals(disagreeText) || vote.equals(naturalText);
```